test/e2e_node/privileged_test.go

```diff
@@ -151,7 +150,6 @@ func (config *PrivilegedPodTestConfig) dialFromContainer(containerIP string, con
 	var output map[string]string
 	err = json.Unmarshal([]byte(stdout), &output)
 	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Could not unmarshal curl response: %s", stdout))
-	glog.Infof("Deserialized output is %v", output)
```

Comment:
cc @yujuhong 

The tests that call this log the map if it does not match expectations.  I don't think we want to output this when the tests pass since it clutters up the build log.